test/fixtures/es-module-loaders/importmaps/array-imports.json

```
@@ -0,0 +1,3 @@
+{
+  "imports": []
+}
```

Can you add a line return for all the files you added in fixtures please?
```suggestion
}

```